include/glow/Support/Error.h

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_SUPPORT_ERROR_H
+#define GLOW_SUPPORT_ERROR_H
+
+#include <type_traits>
+
+#include "llvm/Support/Error.h"
+
+namespace glow {
+
+namespace detail {
+/// NOTE This should not be used directly, instead use UNWRAP or TEMP_UNWRAP.
+/// Callable that takes an llvm::Error or llvm::Expected<T> and exits the
+/// program if the Error is not equivalent llvm::Error::success() or the
+/// Expected<T> contains an error that is not equivalent llvm::Error::success()
+/// TODO(jackmontgomery) replace this with a function that will print file and
```

Comment:
@rdzhabarov what is the convention around TODOs, is this good or should I change it?

Reply:
just, TODO is ok, no name :)